### PR TITLE
Fixes from decisions & addition of /map directory files

### PR DESCRIPTION
**Pull request by Ramit110**

### CODE REVIEW

File: maps/map_types.cwt [hunk 1]
```diff
@@ -0,0 +1,117 @@
+types = {
+    type[province_id] = {
+        path = "game/map"
+        path_file = "positions.txt"
+        localisation = {
+			## optional
```
Comment (Ramit110) [inline maps/map_types.cwt]: Make it required?

File: history/characters.cwt [hunk 2]
```diff
@@ -214,7 +214,7 @@ alias[character_history_effect:raise_levies] = {
     dismiss = bool
 
     ### Where to raise/dismiss the levies.
-    location = enum[province]
+    location = enum[provinces]
```
Comment (Ramit110) [inline history/characters.cwt:217]: AHA! gotcha!

**Comment by tboby** [comment]
LGTM. Did that enum not give an error or put something in the log?

**Comment by Ramit110** [comment]
There were errors yes. Strangely enough.

**Pull request merged**